### PR TITLE
Add machinery helper to release script3

### PR DESCRIPTION
**Pull request by thardeck**

### CODE REVIEW

File: docs/Integration-Test-Setup.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ If osc was already used before the only important part is the addition of `openS
 
 Since this question can't be confirmed in the build script `openSUSE:13.1` needs to be added manually if the repository wasn't used before. The separation char for `trusted_prj` is a space.
 
-The configuration can be tested by running `rake rpm:build` in the git tree.
+The configuration can be tested by running `rake build:machinery` in the git tree.
```
Comment (mauromorales) [inline docs/Integration-Test-Setup.md]: Just a reminder that the Release Process wiki in Alfred would also need to be updated

Comment (mauromorales) [inline docs/Integration-Test-Setup.md]: Same with the Test Environment wiki page

**Comment by mauromorales** [comment]
lgtm, just as a reminder, the `Release Process` and `Test Environment` wiki pages in Alfred would also need to be updated

**Comment by keberle94** [comment]
lgtm

**Pull request merged**